fsharp.spec

```diff
@@ -103,4 +102,5 @@ rm -rf ${RPM_BUILD_ROOT}%{_prefix}/lib/mono/xbuild
 %{_prefix}/lib/mono/gac/policy.3.3.FSharp.Core/
 %{_prefix}/lib/mono/gac/policy.4.0.FSharp.Core/
 %{_prefix}/lib/mono/gac/policy.4.3.FSharp.Core/
+%{_prefix}/lib/mono/xbuild/Microsoft/VisualStudio/
```

Comment:
The full list of targets locations is as follows:

    %(_prefix)/lib/mono/Microsoft\ F#/v4.0/
    %(_prefix)/lib/mono/Microsoft\ SDKs/F#/3.0/Framework/v4.0/
    %(_prefix)/lib/mono/Microsoft\ SDKs/F#/3.1/Framework/v4.0/
    %(_prefix)/lib/mono/xbuild/Microsoft/VisualStudio/v/FSharp/
    %(_prefix)/lib/mono/xbuild/Microsoft/VisualStudio/v12.0/FSharp/

Cheers
Don